var/spack/repos/builtin/packages/acts-algebra-plugins/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class ActsAlgebraPlugins(CMakePackage):
+    """A portable linear algebra library with multiple backends that is part of
+    the Acts ecosystem."""
+
+    homepage = "https://github.com/acts-project/algebra-plugins"
+    url = "https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.24.0.tar.gz"
+    list_url = "https://github.com/acts-project/algebra-plugins/tags"
+
+    maintainers("stephenswat")
+
+    license("MPL-2.0")
+
+    version("0.24.0", sha256="f44753e62b1ba29c28ab86b282ab67ac6028a0f9fe41e599b7fc6fc50b586b62")
+
+    variant(
+        "cxxstd",
+        default="17",
+        values=("17", "20", "23"),
+        multi=False,
+        description="C++ standard used",
+    )
+    variant("eigen", default=False, description="Enables the Eigen plugin")
+    variant("smatrix", default=False, description="Enables the SMatrix plugin")
```

Comment:
Is the `fastor` plugin not desirable?

Reply:
Right, `fastor` wasn't there yet; added in #44984.

Reply:
Would you like to lift the fastor dependency into a new PR or simply update this in #44984?

Reply:
I think #44984 should get merged soon (it's already on auto-merge and only held up by an unrelated CI issue). So, let's keep it for a new PR.

Reply:
I have added Fastor support to this PR!